src/Snapshot.php

```diff
@@ -22,12 +22,22 @@ class Snapshot
     /** @var string */
     public $name;
 
+    /** @var string */
+    public $compressionExt = null;
```

Comment:
In general, I do not like abbreviations. Rename this to `compressionExtension`

Reply:
👍